src/front-end/src/components/AccountMenuItem/AccountMenuItem.js

```diff
@@ -17,7 +17,12 @@ function AccountMenuItem() {
   });
 
   const onLogout = () => {
-    fire.auth().signOut();
+    fire
+      .auth()
+      .signOut()
+      .then(() => {
+        window.location.href = '/my-account';
+      });
```

Comment:
To fix the bug of being left on the same page after logging out:

Reply:
Nice! That was annoying me :)